lib/ansible/modules/crypto/luks_device.py

```diff
@@ -64,6 +64,11 @@
             - "BEWARE that working with keyfiles in plaintext is dangerous.
               Make sure that they are protected."
         type: path
+    keysize:
+        description:
+            - "Sets the key size only if LUKS container does not exist."
+        type: int
+        version_added: '2.9'
```

Comment:
This needs to be changed now:
```suggestion
        version_added: '2.10'
```